coding_sheets/extractor_script.py

```diff
@@ -0,0 +1,330 @@
+import pandas as pd
+import json
+from django.conf import settings
+import coding_sheets.coding_info as coding_info
+
+pd.set_option("display.max_colwidth", 100)
```

Comment:
What does this do/add?

Reply:
This had been added to help in displaying data in the terminal. I've removed it since we no longer need it.